Update README (#6)
aejjehint/oneAPI-samples: DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/cholesky_inversion/README.md

> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
>
> :warning: Make sure you add the device files associated with the FPGA that you are targeting to your Intel® Quartus® Prime installation.

> >
> > :warning: This code sample may fail to compile using Intel® oneAPI DPC++/C++
> > Compiler 2024.2 on Ubuntu 22 and RHEL 8 due to a known bug which will be fixed
> > in the 2024.2 patch. Users who wish to compile this code sample on one of
> > these OSes must make sure to have the 2024.2 patch of the oneAPI compiler
> > installed. 

Reviewer: ```suggestion
>
> :warning: This code sample fails to compile with the Intel® oneAPI DPC++/C++
> Compiler 2024.2 on Ubuntu 22 and RHEL 8 due to a known bug which will be fixed
> in the 2024.2 patch. Users who wish to compile this code sample on one of
> these OSes must make sure to have the 2024.2 patch of the oneAPI DPC++/C++ compiler
> installed. 
```

use stronger language: 'may fail to compile' makes it sounds like it 'may successfully compile'.

don't the patch numbers usually have a name like 2024.2.1 or something? Ideally we would be able to link to the download page for the patch (so that people looking for it in August can find it easily), but the least we can do is include the correct compiler patch version.

Can we state which operating systems that we know can compile the sample successfully? (e.g. SLES15/Windows, ???)
